### PR TITLE
Support the SLA Live Report & Add Overview Diagram

### PR DESCRIPTION
**Pull request by haoel**
exposeing an HTTP server to access the Live SLA report

- HTML: http://localhost:8181/
- JSON: http://localhost:8181/api/v1/sla/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -542,6 +552,13 @@ notify:
 ```YAML
 # Global settings for all probes and notifiers.
 settings:
+
+  # A HTTP Server configuration
+  http:
+    ip: localhost # the IP address of the server. default:""
+    port: 8181 # the port  of the server. default: 8181
```
Comment (zhao-kun) [inline README.md:559]:
```suggestion
    port: 8181 # the port of the server. default: 8181
```

File: README.md [hunk 2]
```diff
@@ -542,6 +552,13 @@ notify:
...
+    refresh: 5s # the auto refresh interval of the server. default: the minimum value of the probes' interval.
```
Comment (zhao-kun) [inline README.md:560]:
```suggestion
    refresh: 5s # the auto-refresh interval of the server. default: the minimum value of the probes' interval.
```

File: README.md [hunk 3]
```diff
@@ -542,6 +552,13 @@ notify:
 ```YAML
 # Global settings for all probes and notifiers.
 settings:
+
+  # A HTTP Server configuration
+  http:
+    ip: localhost # the IP address of the server. default:""
```
[the remark this answers is not in the document]
Comment (zhao-kun) [reply README.md:558]: or default is `0.0.0.0` (all interfaces)?

[the remark this answers is not in the document]
Comment (haoel) [reply README.md]: make the `0.0.0.0` as the default.

**Review by zhao-kun — APPROVED** [review]

**Pull request merged**